migrations/sql/V2020.09.22.15.30__added_now_technical_review_template.sql

```diff
@@ -0,0 +1,9 @@
+INSERT INTO document_template
+    (document_template_code, template_file_path, active_ind, create_user, update_user, form_spec_json)
+VALUES
+    ('NTR', 'templates/now/NOW Technical Review.docx', true, 'system-mds', 'system-mds', '[]');
+
```

Comment:
Is there any issues with the file path due to spaces in the name? Should there be a - or _?

Reply:
Nope :)